Probesys/bileto
# Misc

## Related issue(s)



N/A

## Changes



Review commit by commit.

## How to test manually



- Check that all the organizations pages have an adequate breadcrumb
- Click on "settings" and check that you're redirected to the first page you have access to (orga, roles, or users, based on your authorizations)
- Create an admin role with no permissions, assign it to a user and login with him → check that settings page warn you about the impossible access
- In French, check the gender of impact weight is correct

## Checklist
